.github/workflows/run_tartan.yml

```diff
@@ -0,0 +1,98 @@
+# Build, Run and Test Tartan EchoSystem
+name: Build and Run Tartan
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/run_tartan.yml'
+  schedule:
+    - cron: '0 0 * * *'  # Runs the workflow at midnight every day
+
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Does this need the trailing colon? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

Also maybe delete the two blank lines above as I got a bit confused about the scope